### DISCUSSION TITLE
Support for nodejs v20 LTS

### DISCUSSION THREAD

**Discussion by rconjoe** [vercel/community#4527] [Feature Question] [answered]
### Summary

nodejs v20 becomes active LTS in 6 days. our team is fully prepared for this with the only exception being that vercel runners only support up to 18.x. It looks like there were users facing [a similar situation](https://github.com/orgs/vercel/discussions/37) for previous versions as well.

### Steps to Reproduce

Have a project that requires nodejs v20.

**Comment by rconjoe** [comment]
Is v20 support something we can expect to see in 2023?

**Reply by leerob** [reply]
Yes, we are planning imminent support in beta.

**Comment by leerob** [comment, marked as answer]
Vercel now supports Node.js 20!

https://twitter.com/vercel_changes/status/1725302319695213001
